test/tools/tags/tag-browser.test.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable import/no-unresolved */
+import { expect } from '@esm-bundle/chai';
+import sinon from 'sinon';
+import '../../../tools/tags/tag-browser.js';
+
+const tags = [
+  {
+    path: '/content/cq:tags/audience.1.json',
+    activeTag: '',
+    name: 'audience',
+    title: 'User Guide Audience',
+  },
+  {
+    path: '/content/cq:tags/authoring.1.json',
+    activeTag: '',
+    name: 'authoring',
+    title: 'Authoring',
+  },
+  {
+    path: '/content/cq:tags/caas.1.json',
+    activeTag: '',
+    name: 'caas',
+    title: 'CaaS',
+  },
+];
+
+const ogLana = window.lana;
+
+const delay = (milliseconds) => new Promise((resolve) => { setTimeout(resolve, milliseconds); });
+
+const init = () => {
+  const daTagBrowser = document.createElement('da-tag-browser');
+  document.body.append(daTagBrowser);
+  daTagBrowser.rootTags = tags;
+  daTagBrowser.actions = { sendText: sinon.spy() };
+  daTagBrowser.getTags = async (tag) => tags.map((t) => {
+    t.activeTag = `${tag.activeTag}/${tag.name}`;
+    return t;
+  });
+  return daTagBrowser;
+};
+
+describe('Locale Selector', () => {
+  beforeEach(async () => {
+    document.body.innerHTML = '';
+    window.lana = { log: sinon.spy() };
+  });
+
+  afterEach(() => {
+    window.lana = ogLana;
+  });
```

Comment:
setting up lana and a spy, but I don't think I see you use it in an `expect`?

Reply:
Setting this up as boilerplate since we will want to log to lana in the future.